CMakeLists.txt

```diff
@@ -66,9 +72,14 @@ endif()
 # Set up the build for the LLVM/MLIR git-submodule
 include(cmake/llvm-project.cmake)
 
-# Update the build-tree RPATH
-set(CMAKE_BUILD_RPATH "${ROCMLIR_LIB_DIR};${LLVM_EXTERNAL_LIB_DIR}")
-message(STATUS "CMAKE_BUILD_RPATH: ${CMAKE_BUILD_RPATH}")
+if(MSVC)
```

Comment:
Hold on just a moment - why isn't this stuff being pulled in from upstream's build setup? LLVM builds on Windows, after all?

Also, if this is all duplicative of upstream, can it be moved to `mlir/CMakeLists.txt`?